# structure of the message

float64 x       # x position
float64 y       # y position
float64 vel_x   # linear velocity on the x direction
float64 vel_z   # angular velocity around the z-axis
